consensus/src/selection/step.rs

```diff
@@ -60,9 +61,25 @@ impl<T: Operations + 'static, D: Database> Selection<T, D> {
         committee: Committee,
     ) -> Result<Message, ConsensusError> {
         if committee.am_member() {
+            let iteration = cmp::min(
+                config::RELAX_ITERATION_THRESHOLD,
+                u8::from_step(ctx.step),
+            );
+
+            // Fetch failed certificates from sv_registry
+            let failed_certificates = ctx
+                .sv_registry
+                .lock()
+                .await
+                .get_nil_certificates(0, iteration as usize);
```

Comment:
should this be `iteration - 1` since `iteration` is the current one?
Of course there wouldn't be any nil certificate for `iteration`, but I think it would be more logically sound.

Reply:
No, it follows the default ranging `self.agreement_reg[from..to]` where `to` is excluded